org.adempiere.base/src/org/compiere/process/ProcessInfoParameter.java

```diff
@@ -282,4 +285,78 @@ public void setParameterName (String ParameterName)
 		m_ParameterName = ParameterName;
 	}
 
+	/**
+	 * Return the value of the parameter as a validated String (all values between commas must be integer)
+	 * @return String
+	 */
+	public String getParameterAsMultipleInt() {
+		return getParameterMultipleInt(m_Parameter);
+	}
+
+	/**
+	 * Return the value of the parameter To as a validated String (all values between commas must be integer)
+	 * @return String
+	 */
+	public String getParameter_ToAsMultipleInt() {
+		return getParameterMultipleInt(m_Parameter_To);
+	}
+
+	/**
+	 * Return the value of the parameter as a validated String (all values between commas must be integer)
+	 * @return String
+	 */
+	private String getParameterMultipleInt(Object param) {

```

Comment:
should just use string here and getParameterAsMultipleInt(m_Parameter) can be changed to getParameterAsMultipleInt(getParameterAsString())

Reply:
done ; I also add a getParameter_ToAsString() method